src/components/build/bricksListPage/bricksListPage.tsx

```diff
@@ -1,7 +1,7 @@
 import "./bricksListPage.scss";
 import React, { Component } from "react";
 import {
-  Box, Grid, FormControlLabel, Radio, RadioGroup, Hidden,
+	Box, Grid, FormControlLabel, Radio, RadioGroup, Hidden,
 } from "@material-ui/core";
```

Comment:
@farmanahmed2007 this page is no longer exist. Don`t work on this page. I will remove it when donw with view all